armi/reactor/tests/test_zones.py

```diff
@@ -80,25 +80,51 @@ def test_addItem(self):
 
         self.assertRaises(AssertionError, zone.addItem, "nope")
 
+    def test_removeItem(self):
+        zone = zones.Zone("test_removeItem", [a.getLocation() for a in self.aList])
+        zone.removeItem(self.aList[0])
+        self.assertNotIn(self.aList[0].getLocation(), zone)
+
+        self.assertRaises(AssertionError, zone.removeItem, "also nope")
+
     def test_addItems(self):
         zone = zones.Zone("test_addItems")
         zone.addItems(self.aList)
         for a in self.aList:
             self.assertIn(a.getLocation(), zone)
 
+    def test_removeItems(self):
+        zone = zones.Zone("test_removeItems", [a.getLocation() for a in self.aList])
+        zone.removeItems(self.aList)
+        for a in self.aList:
+            self.assertNotIn(a.getLocation(), zone)
+
     def test_addLoc(self):
         zone = zones.Zone("test_addLoc")
         zone.addLoc(self.aList[0].getLocation())
         self.assertIn(self.aList[0].getLocation(), zone)
 
         self.assertRaises(AssertionError, zone.addLoc, 1234)
 
+    def test_removeLoc(self):
+        zone = zones.Zone("test_removeLoc", [a.getLocation() for a in self.aList])
+        zone.removeLoc(self.aList[0].getLocation())
+        self.assertNotIn(self.aList[0].getLocation(), zone)
+
+        self.assertRaises(AssertionError, zone.removeLoc, 1234)
+
     def test_addLocs(self):
         zone = zones.Zone("test_addLocs")
         zone.addLocs([a.getLocation() for a in self.aList])
         for a in self.aList:
             self.assertIn(a.getLocation(), zone)
 
+    def test_removeLocs(self):
+        zone = zones.Zone("test_removeLocs", [a.getLocation() for a in self.aList])
+        zone.removeLocs([a.getLocation() for a in self.aList])
+        for a in self.aList:
+            self.assertNotIn(a.getLocation(), zone)
+
```

Comment:
Add a test that tries to remove a location not in the Zone

Reply:
This was already done here:
https://github.com/terrapower/armi/blob/ffda4f1850f0835f4f1d95f383bea9dd1f4fdbde/armi/reactor/tests/test_zones.py#L109-L114

But I also added some lines here to test the removal of `Zone`s themselves just now:
https://github.com/terrapower/armi/pull/857/files#diff-d254253f09fc7f48ba48d6dd1a59c74de1f9f5d96437b1d31aff1c0075afbaf3R213-R220